packages/hello-gsm/public/data/faq.json

```
@@ -1,11 +1,11 @@
 [
   {
     "question": "원서접수에 필요로 하는 자격증이 있나요?",
-    "answer": "자격증은 점수에 반영되지 않습니다. 원서접수에 관한 자세한 사항은 2023학년도 모집요강을 참고하시기 바랍니다."
+    "answer": "자격증은 점수에 반영되지 않습니다. 원서접수에 관한 자세한 사항은 2024학년도 모집요강을 참고하시기 바랍니다."
   },
   {
     "question": "원서접수 마감 시간은 언제인가요?",
-    "answer": "접수마감은 전형요강에 명시된 날짜 17:00 까지 이며, 모든 증빙서류가 저희 학교에 도착하여야 합니다. 2023학년도 신입생 원서접수기간 2022.10.22.(월) ~ 10.26.(금) 09:00~17:00"
+    "answer": "접수마감은 전형요강에 명시된 날짜 17:00 까지 이며, 모든 증빙서류가 저희 학교에 도착하여야 합니다. 2024학년도 신입생 원서접수기간 2023.10.16.(월) ~ 10.19.(금) 09:00~17:00"
```

```suggestion
    "answer": "접수마감은 전형요강에 명시된 날짜 17:00 까지 이며, 모든 증빙서류가 저희 학교에 도착하여야 합니다. 2024학년도 신입생 원서접수기간 2023.10.16.(월) ~ 10.19.(목) 09:00~17:00"
```

요일 작성이 잘못되어 있는 것 같습니다!

---

확인 감사합니다 😊